sklearn/decomposition/nmf.py

```diff
@@ -176,6 +252,16 @@ def _nls_subproblem(V, W, H, tol, max_iter, sigma=0.01, beta=0.1):
     max_iter : int
         Maximum number of iterations before timing out.
 
+    regul : double, default: 0.
```

Comment:
This should be `alpha`, but there's already an `alpha` with the meaning of step size in the solver.  I'd suggest the renaming, but this is a private and deprecated function, so I'm ambivalent.